package.json

```diff
@@ -37,11 +39,11 @@
     "ibm_db": "2.7.4"
   },
   "peerDependencies": {
-    "@zowe/imperative": "^4.0.0",
-    "@zowe/cli": "^6.0.0"
+    "@zowe/cli": "^6.0.0",
+    "@zowe/imperative": "^4.0.0"
   },
   "devDependencies": {
-    "@types/fs-extra": "^9.0.5",
+    "@types/fs-extra": "^5.0.5",
```

Comment:
Downgraded `@types/fs-extra` for various reasons:
- the test-tsconfig.json cannot properly exclude the `@types/fs-extra` package since the TestEnvironment.ts uses if for `mkdirpSync`
- tsconfig.jsonhas to include it's node_modules/typings. See https://stackoverflow.com/a/67512016/16555983
- `@types/fs-extra@^9.0.5` has issues that we may be able to avoid with <br/> `tsconfig.json#compilerOptions.skipLibCheck: true`

Note: We may have to upgrade to newer versions of this package for security reasons.

Reply:
Interesting, wasn't sure if `@types/*` packages have vulnerabilities? https://snyk.io/vuln/npm:%40types%2Ffs-extra

Reply:
Seems like no vulnerabilities based on the snyk report 😋 
Still, I think we should eventually update it, which means we will need to address the underlying problem here:
`test-tsconfig not able to exclude the `@types` package from the TestEnvoromnet.`

We can take the workaround of `skipLibCheck: true` or try to replace the `mkdirpSync` functionality 😋